src/content/docs/guides/usage/data/shape-data.mdx

```diff
@@ -121,6 +121,27 @@ drop x
 {y: "baz"}
 ```
 
+## Relocate fields with `move`
+
+Use the `move` operator to combine `set` and `drop`. This pipeline
+
+```tql
+from {old: 42}
+move new = old
+```
+
+produces the same output as this one:
+
+```tql
+from {old: 42}
+set new = old
+drop old
+```
+
+```tql
+{new: 42}
+```
```

Comment:
Do you think it'd make sense to add an example here that moves multiple fields to explain that this:

```
from {foo: 1, bar: 2}
move foo=bar, qux=foo
```

Is equivalent to this:

```
from {foo: 1, bar: 2}
set foo=bar, qux=foo
drop bar, foo
```

But not equivalent to this:

```
from {foo: 1, bar: 2}
set foo=bar
drop bar
set qux=foo
drop foo
```

Reply:
Added it in 19560c3.